src/components/IncidentDetails/DefinitionOfDone/IncidentsDefinitionOfDoneItem.tsx

```diff
@@ -0,0 +1,98 @@
+import { Menu } from "../../Menu";
+import { AiFillCheckCircle } from "react-icons/ai";
+import { BsHourglassSplit, BsTrash } from "react-icons/bs";
+import { Evidence } from "../../../api/services/evidence";
+import { IconButton } from "../../IconButton";
+import { EvidenceView } from "./EvidenceView";
+import { Dispatch, SetStateAction, useState } from "react";
+import { Size } from "../../../types";
+import { FaEdit } from "react-icons/fa";
+import EditEvidenceDefinitionOfDoneScript from "./EditEvidenceDefinitionOfDoneScript";
+
+type Props = {
+  evidence: Evidence;
+  setOpenDeleteConfirmDialog: Dispatch<SetStateAction<boolean>>;
+  setEvidenceBeingRemoved: Dispatch<SetStateAction<Evidence | undefined>>;
+  refetch: () => void;
+};
+
+export default function IncidentsDefinitionOfDoneItem({
+  evidence,
+  setEvidenceBeingRemoved,
+  setOpenDeleteConfirmDialog,
+  refetch
+}: Props) {
+  const [isEditingDoDScriptModalOpen, setIsEditingDoDScriptModalOpen] =
+    useState(false);
+
+  return (
+    <>
+      <div className="relative flex items-center py-2">
+        {evidence.done ? (
+          <AiFillCheckCircle className="mr-1" />
+        ) : (
+          <BsHourglassSplit className="mr-1" />
+        )}
+        <div className="min-w-0 flex-1 text-sm ml-2">
+          <EvidenceView evidence={evidence} size={Size.small} />
+        </div>
+        <div className="flex items-center">
+          <Menu>
+            <Menu.VerticalIconButton />
+            <Menu.Items widthClass="w-72">
+              <Menu.Item
+                onClick={(e: any) => {
+                  setIsEditingDoDScriptModalOpen(true);
+                }}
+              >
+                <div className="cursor-pointer flex w-full">
+                  <FaEdit />
+                  <span className="pl-2 text-sm block cursor-pointer">
+                    Edit Definition of done Script
+                  </span>
+                </div>
+              </Menu.Item>
+              <Menu.Item
+                onClick={(e: any) => {
+                  setOpenDeleteConfirmDialog(true);
+                  setEvidenceBeingRemoved(evidence);
+                }}
+              >
+                <div className="cursor-pointer flex w-full">
+                  <IconButton
+                    className="bg-transparent flex items-center"
+                    ovalProps={{
+                      stroke: "blue",
+                      height: "18px",
+                      width: "18px",
+                      fill: "transparent"
+                    }}
+                    icon={
+                      <BsTrash
+                        className="text-gray-600 border-0 border-l-1 border-gray-200"
+                        size={18}
+                      />
+                    }
+                  />
+                  <span className="pl-2 text-sm block cursor-pionter">
+                    Remove from Definition of done
```

Comment:
I am not expecting styling to be added here, it should be in the IconButton itself.